src/main/java/frc/robot/commands/MoveMovableSubsystem.java

```diff
@@ -0,0 +1,39 @@
+package frc.robot.commands;
+
+import java.util.function.BooleanSupplier;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.MoveableSubsystem;
+
+public class MoveMovableSubsystem extends CommandBase {
+  private MoveableSubsystem subsystem;
+  private BooleanSupplier isFinished;
+  private double power;
+
+  /** This command can move any subsystem that implements MovableSubsytem. */
```

Comment:
/** This command can move any subsystem that implements MovableSubsytem. */
this constructor uses BooleanSupplier for... etc.